### DISCUSSION TITLE
backdrop-filter with '-webkit' prefix

### DISCUSSION THREAD

**Discussion by Hengry** [tailwindlabs/tailwindcss#4748] [Ideas]
It seems in backdrop-filter does not add -webkit in backdrop-filter now, so it cannot shows properly in safari.

**Comment by olets** [comment]
Unless you use the Tailwind CLI, browser prefixes are not added. The documentation gives a minimal example of how to add Autoprefixer to your build process (https://tailwindcss.com/docs/browser-support#vendor-prefixes), and the Autoprefixer docs explain how to configure support for the browsers you target.
